app/Admin.cpp

```diff
@@ -29,9 +27,6 @@ Admin::Admin(QWidget *parent) :
 	PopulateComboBoxes("SELECT roofType FROM teamInfo"   , ui -> Roof_Type_ComboBox);
 	PopulateComboBoxes("SELECT teamName FROM teamInfo"   , ui -> Team_Name_ComboBox);
 
-	//Connect mouse event to data table
-	//connect(ui -> Update_Souvenir_Datatable, SIGNAL(clicked(const QModelIndex &)), this, SLOT(onTableClicked(const QModelIndex &)));
-
 }
 
 //Populates all combo boxes with relaed information
```

Comment:
line 51: use query.value(0) get ID, query.value(1) to get name
line 58: use combobox.setItemData() to add the ID to the combobox